nixos/lib/make-options-doc/default.nix

```diff
@@ -130,6 +130,8 @@ in rec {
         if baseOptionsJSON == null
         then builtins.toFile "base.json" "{}"
         else baseOptionsJSON;
+
+      NIXPKGS_PATH = pkgs.path;
```

Comment:
@roberth should this be `toString`ed or something to avoid copying?

Reply:
I don't think you need all of Nixpkgs. Could you pass the urls json instead?

Reply:
Yeah.